packages/common/src/core/_internal/PropertyMapper.ts

```diff
@@ -88,6 +89,19 @@ export function mapObjectToModel<T>(
   model: IModel,
   mappings: IPropertyMap[]
 ): IModel {
+  if (model.item?.typeKeywords) {
```

Comment:
Do we need `hasOwnProperty` and `getProp` here? See my below comment about reusing the `isDiscussable` utility currently in `hub-discussions`. I'm curious if this could be simplified to just:

```jsx
if (model.item?.typeKeywords) {
  model.item.typeKeywords = object.isDiscussable
    ? model.item.typeKeywords.filter(typeKeyword => typeKeyword !== CANNOT_DISCUSS)
    : [...model.item.typeKeywords, CANNOT_DISCUSS];
}
```